app/Config/core.php

```diff
@@ -215,19 +215,19 @@
  * the cake shell command: cake schema create Sessions
  *
  */
-	Configure::write('Session', array(
-		'defaults' => 'php'
-	));
+    Configure::write('Session', array(
+        'defaults' => 'php'
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 33 spaces but found 8